pkg/minikube/constants/constants.go

```diff
@@ -206,6 +209,9 @@ func GetKubeadmCachedImages(version string) []string {
 		"gcr.io/google_containers/kube-scheduler-amd64:" + version,
 		"gcr.io/google_containers/kube-controller-manager-amd64:" + version,
 		"gcr.io/google_containers/kube-apiserver-amd64:" + version,
+
+		//Storage provisioner
+		"gcr.io/k8s-minikube/storage-provisioner:" + version,
```

Comment:
I think you can just leave this as whatever is above.  The version here is actually from the `--kubernetes-version` parameter, so if you use `--kubernetes-version v1.8.1` it will try to cache the 1.8.1 tag of the storage provisioner. 

We probably won't publish one of these for every version of kubernetes, so make this the hardcoded 
`"gcr.io/k8s-minikube/storage-provisioner:v1.8.0"`